fuse_constraints/include/fuse_constraints/normal_delta_pose_2d_cost_functor.h

```diff
@@ -123,7 +123,7 @@ bool NormalDeltaPose2DCostFunctor::operator()(
   full_residuals_vector.template head<2>() =
     fuse_core::rotationMatrix2D(orientation1[0]).transpose() * (position2_vector - position1_vector) -
     b_.head<2>().template cast<T>();
-  full_residuals_vector(2) = fuse_core::wrapAngle2D(orientation2[0] - orientation1[0]) - T(b_(2));
+  full_residuals_vector(2) = fuse_core::wrapAngle2D(orientation2[0] - orientation1[0] - T(b_(2)));
```

Comment:
Indeed, this change makes this the same as https://github.com/locusrobotics/fuse/blob/devel/fuse_constraints/src/normal_delta_orientation_2d.cpp